backend/src/main/java/com/google/sps/data/Match.java

```diff
@@ -37,11 +37,11 @@ public long getId() {
     return id;
   }
 
-  public Participant getFirstParticipant() {
+  public com.google.sps.data.Participant getFirstParticipant() {
```

Comment:
Remove "com.google.sps.data." here and below at line 44

Reply:
Ack